src/Omega_h_adios2.cpp

```diff
@@ -0,0 +1,566 @@
+#include "Omega_h_adios2.hpp"
+#include <adios2.h>
+#include <iostream>
+#include "Omega_h_inertia.hpp" //<inertia::Rib>
+#include "Omega_h_timer.hpp"
+#include "Omega_h_tag.hpp"
+#include <map>
+
+using namespace std;
+
+namespace Omega_h {
+
+namespace adios {
+
+long unsigned int comm_size, rank;
+
+template <typename T>
+static void write_value(adios2::IO &io, adios2::Engine &writer, 
+		T val, std::string &name, bool global=false)
+{
+  if (global)
+  {
+    adios2::Variable<T> bpData = io.DefineVariable<T>(name);
+    writer.Put(bpData, val);
+  }
+  else // local
+  {
+    std::vector<T> myData;
+    myData.push_back(val);
+
+    adios2::Variable<T> bpData =
+          io.DefineVariable<T>(name, {comm_size}, {rank},
+          {1}, adios2::ConstantDims);
+    writer.Put(bpData, myData.data());
+  }
+  //std::cout <<__func__<<" "<<name<<": value "<<val<<"\n";
```

Comment:
Please remove all code that is commented out.  There are other locations below; I didn't add a comment on them.